src/main/java/com/minecolonies/coremod/client/gui/WindowScan.java

```diff
@@ -6,18 +6,22 @@
 import com.minecolonies.api.util.constant.Constants;
 import com.minecolonies.blockout.Color;
 import com.minecolonies.blockout.Pane;
+import com.minecolonies.blockout.controls.Button;
 import com.minecolonies.blockout.controls.ItemIcon;
 import com.minecolonies.blockout.controls.Label;
 import com.minecolonies.blockout.controls.TextField;
 import com.minecolonies.blockout.views.ScrollingList;
 import com.minecolonies.coremod.MineColonies;
-import com.minecolonies.coremod.entity.EntityCitizen;
+import com.minecolonies.coremod.network.messages.RemoveBlockMessage;
+import com.minecolonies.coremod.network.messages.RemoveEntityMessage;
 import com.minecolonies.coremod.network.messages.ScanOnServerMessage;
 import net.minecraft.block.Block;
+import net.minecraft.block.BlockLiquid;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'net.minecraft.block.BlockLiquid'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AUselessImportCheck)